src/Component/WeatherForecast.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react';
+
+import clear from '../img/weather-icons/clear.svg';
+import cloudy from '../img/weather-icons/cloudy.svg';
+import drizzle from '../img/weather-icons/drizzle.svg';
+import fog from '../img/weather-icons/fog.svg';
+import mostlycloudy from '../img/weather-icons/mostlycloudy.svg';
+import partlycloudy from '../img/weather-icons/partlycloudy.svg';
+import rain from '../img/weather-icons/rain.svg';
+import snow from '../img/weather-icons/snow.svg';
+import storm from '../img/weather-icons/storm.svg';
+import unknown from '../img/weather-icons/unknown.svg';
+
+class WeatherForecast extends Component {
+  render() {
+    return (
+        <div className="app_main">
+        <div className="twentyfour_hours  md-col-7 ">
+          <div>
+            <div className="time">03:00</div>
```

Comment:
Again, can you use the props you passed down here?

Reply:
If you're not already using it - install the Google Chrome extension 'React Dev Tools'. It will show you what state / props each of your components is receiving.